sparse/tests/test_core.py

```diff
@@ -392,6 +416,66 @@ def test_scalar_exponentiation():
         assert_eq(x ** -1, a ** -1)
 
 
+def test_bool_and():
+    x = random_x_bool((2, 3, 4))
+    xx = COO.from_numpy(x)
+
+    y = random_x_bool((2, 3, 4))
+    yy = COO.from_numpy(y)
+
+    assert_eq(x & y, xx & yy)
+
+
+def test_int_and():
+    x = random_x((2, 3, 4), dtype=int)
+    xx = COO.from_numpy(x)
+
+    y = random_x((2, 3, 4), dtype=int)
+    yy = COO.from_numpy(y)
+
+    assert_eq((x & y), (xx & yy))
+
+
+def test_bool_or():
+    x = random_x_bool((2, 3, 4))
+    xx = COO.from_numpy(x)
+
+    y = random_x_bool((2, 3, 4))
+    yy = COO.from_numpy(y)
+
+    assert_eq(x | y, xx | yy)
+
+
+def test_int_or():
+    x = random_x((2, 3, 4), dtype=int)
+    xx = COO.from_numpy(x)
+
+    y = random_x((2, 3, 4), dtype=int)
+    yy = COO.from_numpy(y)
+
+    assert_eq(x | y, xx | yy)
+
+
+def test_bool_xor():
+    x = random_x_bool((2, 3, 4))
+    xx = COO.from_numpy(x)
+
+    y = random_x_bool((2, 3, 4))
+    yy = COO.from_numpy(y)
+
+    assert_eq(x ^ y, xx ^ yy)
+
+
+def test_int_xor():
+    x = random_x((2, 3, 4), dtype=int)
+    xx = COO.from_numpy(x)
+
+    y = random_x((2, 3, 4), dtype=int)
+    yy = COO.from_numpy(y)
+
+    assert_eq(x ^ y, xx ^ yy)
```

Comment:
It looks like these tests can be merged using `pytest.mark.parametrize('op', [operator.xor, operator.or_, operator.and_, ...])`

Reply:
I'll have a look at it next week.